src/ui/Querybox/Querybox.ts

```diff
@@ -152,84 +152,48 @@ export class Querybox extends Component {
 
     /**
      * Whether to convert a basic expression containing at least a certain number of keywords (see the
-     * [`partialMatchKeywords`]{@link Querybox.options.partialMatchKeywords} option) to a *partial match expression*, so
+     * [`partialMatchKeywords`]{@link Querybox.options.partialMatchKeywords} option) to *partial match expression*, so
      * that items containing at least a certain number of those keywords (see the
      * [`partialMatchThreshold`]{@link Querybox.options.partialMatchThreshold} option) will match the expression.
      *
      * **Notes:**
+     * - Only the basic expression of the query (see [`q`]{@link q}) can be converted to a partial match expression.
+     * - When the [`enableQuerySyntax`]{@link Querybox.options.enableQuerySyntax} option is set to `true`, this feature has no effect if the basic expression contains advanced query syntax (field expressions, operators, etc.).
      *
-     * > - Only the basic expression of the query (see [`q`]{@link q}) can be converted to a partial match expression.
-     * > - When the [`enableQuerySyntax`]{@link Querybox.options.enableQuerySyntax} option is set to `true`, this
-     * > feature has no effect on a basic expression containing query syntax (field expressions, operators, etc.).
-     *
-     * **Example:**
-     *
-     * > With the following markup configuration, if a basic expression contains at least 4 keywords, items containing
-     * > at least 75% of those keywords (round up) will match the query.
-     * > ```html
-     * > <div class='CoveoQuerybox' data-enable-partial-match='true' data-partial-match-keywords='4' data-partial-match-threshold='75%'></div>
-     * > ```
-     * > For instance, if the basic expression is `Coveo custom component configuration help`, items containing
-     * > all 5 of those keywords, or 4 of them (75% of 5 rounded up) will match the query.
-     *
-     * Default value is `false`, which implies that an item must contain all of the basic expression keywords to match
-     * the query.
      * @notSupportedIn salesforcefree
      */
     enablePartialMatch: ComponentOptions.buildBooleanOption({ defaultValue: false }),
 
     /**
-     * The minimum number of keywords that need to be present in a basic expression to convert it to a partial match
-     * expression.
+     * The minimum number of keywords that need to be present in the basic expression to convert it to a partial match expression.
      *
      * See also the [`partialMatchThreshold`]{@link Querybox.options.partialMatchThreshold} option.
      *
      * **Notes:**
-     * > - This option has no meaning unless the [`enablePartialMatch`]{@link Querybox.options.enablePartialMatch}
```

Comment:
Is this point no longer applicable? Or is it already being communicated somewhere else?

Reply:
It is still applicable, but it is now redundant since TypeDoc will use the "depend" property of the buildNumberOption call's argument to generate a tag:

<img width="512" alt="Screen Shot 2020-02-03 at 2 42 02 PM" src="https://user-images.githubusercontent.com/23503066/73685030-5fd2aa80-4693-11ea-818a-6208ba873e57.png">